images/linux/toolsets/toolset-2004.json

```diff
@@ -153,7 +153,13 @@
         ]
     },
     "pipx": [
-        "yamllint",
-        "aws-sam-cli"
+        {
+            "package": "yamllint",
+            "python": "default"
+        },
+        {
+            "package": "aws-sam-cli",
+            "python": "/opt/hostedtoolcache/Python/3.7*/x64/bin/python3.7"
```

Comment:
I suggest to put only `3.7` to toolset and build the remain line in code.
Also, you can just skip `python`  if it is a default

Reply:
Done